File: OLSource1/v1/VS_officedev/codesnippet/CSharp/programmatic-limitations-of-host-items-and-host-controls_1.cs
            this.Range["A1"].Value2 = "Monday";
            this.Range["A2"].Value2 = "Tuesday";

            Microsoft.Office.Tools.Excel.NamedRange dayRange = 
                this.Controls.AddNamedRange(this.Range["A1", "A7"], "dayRange");
            this.Range["A1", "A2"].AutoFill(dayRange.InnerObject, Excel.XlAutoFillType.xlFillDays);

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/programmatic-limitations-of-host-items-and-host-controls_1.vb
        Me.Range("A1").Value2 = "Monday"
        Me.Range("A2").Value2 = "Tuesday"

        Dim dayRange As Microsoft.Office.Tools.Excel.NamedRange = _
            Me.Controls.AddNamedRange(Me.Range("A1", "A7"), "dayRange")
        Me.Range("A1", "A2").AutoFill(dayRange.InnerObject, Excel.XlAutoFillType.xlFillDays)